src/main/resources/templates/restaurantprofile.html

```diff
@@ -49,16 +49,10 @@ <h3>Contact Customer</h3>
         })
 		const responseData = await response.json();
 		console.log(responseData);
+        document.getElementById('restaurantName').textContent = responseData.restaurantName+'\'s Restaurant Profile';
     }
     
-//  logout() {
-//  localStorage.removeItem('user');
-//  localStorage.removeItem('token');
-//  window.location.href = 'http://localhost:8080/';
-//}
-    
```

Comment:
I put this here because I think a logout button should be in the header of every page once the user is logged in.  This function would run when that button is clicked